packages/nextjs/utils/scaffold-eth/notification.tsx

```diff
@@ -49,14 +49,14 @@ const Notification = ({
   return toast.custom(
     t => (
       <div
-        className={`flex flex-row items-center justify-between max-w-sm bg-white p-4 text-black shadow-2xl hover:shadow-none transform-gpu translate-y-0 hover:translate-y-1 rounded-md relative transition-all duration-500 ease-in-out space-x-2 ${
+        className={`flex flex-row items-start justify-between max-w-sm bg-base-200 p-4 shadow-2xl hover:shadow-none transform-gpu translate-y-0 hover:translate-y-1 rounded-md relative transition-all duration-500 ease-in-out space-x-2 ${
           t.visible ? "top-0" : "-top-96"
```

Comment:
Why do you need the -top-96 when not visible?

Reply:
I didn't notice it till you pointed it out.
I just checked, it seems that "-top-96" position helps create the slide up out of the screen animation, without it the notification just disappears in place but this brings up two points what about notifications with position "bottom-xxx"?
- In the case where notification position is bottom, when the notification is about to disappear it moves up to somewhere in the middle of the screen and seems to look like a glitch (the screenshot bellow explains it better) (will fix this in my next commit)

From here:
![image](https://user-images.githubusercontent.com/62877695/231141976-33ce32c6-b0a9-411c-a204-a6c7bd74f81b.png)

To here:
![image](https://user-images.githubusercontent.com/62877695/231142024-ac07bce1-2ca4-443c-b08d-6f5d25ddd848.png)

- When hovering on the notification the shadow is set to none and the notification moves down by one ("translate-y-1") this animation looks weird when notification position is bottom, I think it should have a mirror effect and the notification should move up by 1 ("translate-x-1"). Would love to have your opinion on this before I change it.

Reply:
Maybe we can check how it feels if the notification is at the top.

Reply:
I didn't get you there

Reply:
Sorry. 

About "When hovering on the notification the shadow is set to none and the notification moves down by one ("translate-y-1") this animation looks weird when the notification position is bottom, I think it should have a mirror effect and the notification should move up by 1 ("translate-x-1"). Would love to have your opinion on this before I change it."

Maybe the animation looks good when the notification is at the top, so if you change it to "translate-x-1", now the notification at the top will look weird.

Please, let me know if I'm not clear enough.

Reply:
Done